ni_measurementlink_service/_internal/discovery_client.py

```diff
@@ -260,11 +260,15 @@ def _key_file_exists(key_file_path: pathlib.Path) -> bool:
 def _start_service(exe_file_path: pathlib.PurePath, key_file_path: pathlib.Path) -> None:
     """Starts the service at the specified path and wait for the service to get up and running."""
     global _discovery_service_subprocess  # save Popen object to avoid ResourceWarning
+
+    # Added hexadecimal equivalent of subprocess.CREATE_BREAKAWAY_FROM_JOB to avoid
+    # Mypy error 'Module subprocess has no attribute CREATE_BREAKAWAY_FROM_JOB'.
```

Comment:
The reason for this error is that `CREATE_BREAKAWAY_FROM_JOB` is Windows-specific:
- https://github.com/python/cpython/blob/main/Lib/subprocess.py#L80
- https://github.com/python/typeshed/blob/main/stdlib/subprocess.pyi#L2585

In addition, the `creationflags` parameter is only supported on Windows and will cause an error on Linux.
- https://github.com/python/cpython/blob/main/Lib/subprocess.py#L854

Our build workflow runs mypy on Linux, so Windows-specific constants are not available. The solution is to check `sys.platform` before calling platform-specific code.

```
kwargs: Dict[str, Any] = {}
if sys.platform == "win32":
    # Terminating the measurement service should not terminate the discovery service.
    kwargs["creationflags"] = subprocess.CREATE_BREAKAWAY_FROM_JOB

_discovery_service_subprocess = subprocess.Popen(
    ...
    stderr=subprocess.DEVNULL,
    **kwargs
)
```

Reply:
Thanks for the inputs Brad! I have updated the `creationflags` as plaform specific. The same change I have done in tests as well.